### PR TITLE
Update rpm installation instruction

### PR DESCRIPTION
**Pull request by leonjia0112**

### CODE REVIEW

**Comment by leonjia0112** [comment]
How would I resolve the conflict?

**Comment by leonjia0112** [comment]
I feel like the CI needs to install Keylime so it can pass the keylime command test.

**Comment by leonjia0112** [comment]
I don't have the "Merge branch 'master' into branch2" in my local commit log, but I don't know why it is here after I push it.

**Comment by frozencemetery** [comment]
If it's actually not there: you need to force-push the branch.  Otherwise github is showing the same thing.

**Comment by leonjia0112** [comment]
I found out what happened, when I resolve the conflict, there will be an extra commit message.

File: rpm/README.md [hunk 1]
```diff
@@ -35,7 +35,27 @@ Use `sudo rpm -ivh python-keylime-1.2-1.noarch.rpm` to install the RPM. Use
 package using `yum install python-devel`. Once finished, the RPM should
 install with no issues.
 
-### Running Keylime
+### Install Keylime
 
 The RPM installs Keylime to the `usr/bin/python-keylime` directory. Everything
-needed to run Keylime can be found there.
+needed to run Keylime can be found there.Execute Keylime installer to install
```
Comment (mbestavros) [inline rpm/README.md]: Missing a space on this line. Should be:

```suggestion
needed to run Keylime can be found there. Execute Keylime installer to install 
```

File: rpm/README.md [hunk 2]
```diff
@@ -35,7 +35,27 @@ Use `sudo rpm -ivh python-keylime-1.2-1.noarch.rpm` to install the RPM. Use
 package using `yum install python-devel`. Once finished, the RPM should
 install with no issues.
 
-### Running Keylime
+### Install Keylime
 
 The RPM installs Keylime to the `usr/bin/python-keylime` directory. Everything
-needed to run Keylime can be found there.
+needed to run Keylime can be found there.Execute Keylime installer to install 
+`Keylime` and `tmp4720` to
```
Comment (mbestavros) [inline rpm/README.md]: I don't think you need a line break here:

```suggestion
`Keylime` and `tmp4720` to your system.
```

File: rpm/README.md [hunk 3]
```diff
@@ -35,7 +35,27 @@ Use `sudo rpm -ivh python-keylime-1.2-1.noarch.rpm` to install the RPM. Use
...
 install with no issues.
 
-### Running Keylime
+### Install Keylime
 
 The RPM installs Keylime to the `usr/bin/python-keylime` directory. Everything
-needed to run Keylime can be found there.
+needed to run Keylime can be found there.Execute Keylime installer to install 
+`Keylime` and `tmp4720` to 
+your system.
```
Comment (mbestavros) [inline rpm/README.md]: With the above suggestion, this line can be deleted.

```suggestion
```

**Comment by frozencemetery** [comment]
Pushed to master as 11da55494a6ada09ecf01ab5310ed8ebdc98ef2e.  Please take a look at how I have adjusted the commit message to see what I mean.

**Pull request closed without merging**